src/fsharp/FSharp.Core/string.fs

```diff
@@ -104,7 +104,5 @@ namespace Microsoft.FSharp.Core
 
         [<CompiledName("Length")>]
         let length (str:string) =
-            if String.IsNullOrEmpty str then
-                0
-            else
-                str.Length
+            if isNull str then 0
```

Comment:
The change is good. Typically this would be formatted to a single line in this codebase, though. If you make that change I'll approve this.

Reply:
@cartermp, thx, I forgot, @forki mentioned that too. Fixed in https://github.com/dotnet/fsharp/pull/9469/commits/91ca1d4891aed625c5e5f1e821da649015f8acab